docroot/themes/humsci/humsci_basic/src/scss/tools/_functions.animation.scss

```diff
@@ -1,3 +1,26 @@
+// Returns the proper transition based on the theme.
+// Filters through $hb-animation-list array to find the relevant
+// duration and timing values for the theme.
 @function hb-transition($property) {
-  @return $property $hb-animation-duration $hb-animation-timing-function;
+  $is-valid-theme: false;
+
+  @each $theme in $hb-global-theme-list {
+    @if ($theme == $hb-current-theme) {
+      $is-valid-theme: true;
+    }
+  }
+
+  @if ($is-valid-theme) {
+    $theme-animation-list: map-get($hb-animation-list, $hb-current-theme);
+    $hb-animation: $theme-animation-list[];
```

Comment:
can we call this $animation-transition?

Reply:
I think this assignment should also happen under if statement on line 17

Reply:
maybe instead of doing this way, you can make it clearer what's happening and assign different variables to the two values coming out:

```CSS
$timing: nth($theme-animation-list, 1);
$animation-function: nth($theme-animation-list, 2);
```

https://sass-lang.com/documentation/modules/list#length

Reply:
This is awesome! Thank you!! 🏆